libs/datamodel/core/src/transform/ast_to_dml/db/walkers.rs

```diff
@@ -0,0 +1,195 @@
+use super::{
```

Comment:
Feedback on the walker abstraction is very welcome. I thought about alternatives, but it just felt like the better one.